### PR TITLE
Xiaomi Mijia Bluetooth Wireless Temperature and Humidity Sensor documentation

### PR DESCRIPTION
**Pull request by ratcashdev**
realted implementation pull request: https://github.com/home-assistant/home-assistant/pull/13955

### CODE REVIEW

File: source/_components/sensor.mitemp_bt.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: "Xiaomi Mijia BLE Temperature and Humidity sensor with LCD"
+description: "Instructions on how to integrate MiTemp BLE temperature and humidity sensor with Home Assistant."
+date: 2018-04-22 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mitemp.png
```
Comment (syssi) [inline source/_components/sensor.mitemp_bt.markdown]: Please use the brand logo.

File: source/_components/sensor.mitemp_bt.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ha_category: DIY
+ha_release: 0.??
```
Comment (syssi) [inline source/_components/sensor.mitemp_bt.markdown]: Please use 0.68.

File: source/_components/sensor.mitemp_bt.markdown (hunk 1 above)
Comment (ratcashdev) [reply source/_components/sensor.mitemp_bt.markdown]: as you wish.

**Comment by syssi** [comment]
This is a 99% copy of the miflora documentation. This should faciliate the review: https://hastebin.com/uyujugohok.diff

File: source/_components/sensor.mitemp_bt.markdown [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  - platform: mitemp_bt
+    mac: 'xx:xx:xx:xx:xx:xx'
+    monitored_conditions:
+      - temperature
+```
+
+- **mac** (*Required*): The MAC address of your sensor.
+- **monitored_conditions** array (*Optional*): The parameters that should be monitored (defaults to monitoring all parameters).
+  - **temperature**: Temperature at the sensor's location.
+  - **humidity**: Moisture in the soil.
```
Comment (syssi) [inline source/_components/sensor.mitemp_bt.markdown]: Could you update the description? This isn't true anymore. ;-)

**Review by syssi — DISMISSED** [review]

**Review by syssi — APPROVED** [review]

**Pull request merged**